src/resources/ModifierTemplates/ModifierTemplates.ts

```diff
@@ -5,7 +5,7 @@ export default class ModifierTemplates extends Resource {
     static baseUrl = `/rest/modifiertemplates`;
 
     list() {
-        return this.api.get<ModifierModel>(`${ModifierTemplates.baseUrl}`);
+        return this.api.get<ModifierModel[]>(`${ModifierTemplates.baseUrl}`);
```

Comment:
is this not a breaking change? :thinking:

Reply:
Theoretically yes but the call never worked so it was never used. The backend never returned a single model.

Reply:
Ok then :smile:  :sealOfApproval: